lib/Doctrine/ORM/Decorator/EntityManagerDecorator.php

```diff
@@ -275,4 +275,12 @@ public function getCache()
     {
         return $this->wrapped->getCache();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function findInEntityInsertions($entityName, array $criteria, array $orderBy = null)
```

Comment:
Should not be new public API - if we want to provide this sort of API, it should be done just at repository level (part of `findOneBy`), IMO